doc/controls/DrawerFlyoutPresenter.md

```diff
@@ -57,3 +102,34 @@ Example:
 </Button>
 ```
 > note: Here `VisibleBoundsPadding.PaddingMask` is used to prevent the content from being placed outside of the user-interactable area on mobile devices.
+
+### Extended Use Cases
+- Rounded Corner
+    ```xml
+    <Flyout Placement="Full">
+        <Flyout.FlyoutPresenterStyle>
+            <Style BasedOn="{StaticResource BottomDrawerFlyoutPresenterStyle}" TargetType="FlyoutPresenter">
+                <Setter Property="CornerRadius" Value="16,16,0,0" />
+            </Style>
+        </Flyout.FlyoutPresenterStyle>
+        <Border toolkit:VisibleBoundsPadding.PaddingMask="All" Padding="16,16,0,0">
+            <!-- flyout content -->
+        </Border>
+    </Flyout>
+    ```
+    > remarks: `Padding` is used on the flyout content to avoid content being clipped.
```

Comment:
[!NOTE]

Reply:
this is just another btw/additional commentary, dont need a `alert`` tag for it
i mean ive been using this type of thing way before the introduction of docfx